src/Lecture3.hs

```diff
@@ -92,7 +123,13 @@ weekday to the second.
 >>> daysTo Friday Wednesday
 5
 -}
-daysTo = error "TODO"
+daysTo :: Weekday -> Weekday -> Int
+daysTo d1 d2
+  | n2 >= n1 = n2 - n1
+  | otherwise = 7 - (n1 - n2)
+ where
+  n1 = fromEnum d1
+  n2 = fromEnum d2
```

Comment:
You can implement this function as a one-liner if you remember how `mod` works for negative numbers 🙂